Enable go modules usage, including prerequisites (#145)
lytics/grid: README.md

**The work around** is to create a `vendor` directory:

```
go mod vendor
```

> **Note** that `Gopkg.lock`,`Gopkg.toml` and `vendor/`are all included in the `.gitignore` file. 

Reviewer: ```suggestion
**Note** `vendor/`is included in the `.gitignore` file. 
```
